app/models/media_resource.rb

```diff
@@ -5,7 +5,7 @@ class MediaResource < Valhalla::Resource
   include Schema::Common
   attribute :id, Valkyrie::Types::ID.optional
   attribute :member_ids, Valkyrie::Types::Array
-  attribute :member_of_collection_ids
+  attribute :member_of_collection_ids, Valkyrie::Types::Array
```

Comment:
Is it important this is an array?

Reply:
when it's not I have problems adding to it because it gets initialized as nil. maybe using change sets will help? I'll try taking it out once I've got the changeset stuff in place.

Reply:
Oh, that's because the default is a `Valkyrie::Types::Set.optional`. `Valkyrie::Types::Set` will also work.